ebos/ecloutputblackoilmodule.hh

```diff
@@ -307,7 +307,7 @@ public:
         }
 
         if (FluidSystem::phaseIsActive(waterPhaseIdx) && rstKeywords["KRW"] > 0) {
-            rstKeywords["KRW"] = 0;
+	    rstKeywords["KRW"] = 0;
```

Comment:
tabulator instead of spaces? That hunk also seems to be an artifact.

Reply:
This was also unintentional and has been removed.